### DISCUSSION TITLE
Saving a list of youtube URI's to a playlist to load later

### DISCUSSION THREAD

**Discussion by Timmymayes** [isamert/empv.el#90] [General]
So far 99% of what i'm wanting to do works fantastically with empv and its a great package. However the one thing I'd like to do is save an active playlist of youtube links to a file using empv-playlist-save-to-file. This does save the file but I'm not able to load it with empv-play or any other commands. 

Didn't see this in the documentation and wanted to check if its possible.

**Comment by isamert** [comment]
If you save the playlist with `.m3u` extension, then empv should be able to play it with `empv-play-file`.

I don't know any way to make `read-file-name` enforce a suffix but I can add the following to the save prompt `(extension must be .m3u)` to indicate this. What do you think?
